R/plumber.R

```diff
@@ -145,14 +145,18 @@ Plumber <- R6Class(
     #' This value does not need to be explicitly assigned. To explicitly set it, see [options_plumber()].
     #' @param debug Deprecated. See `$setDebug()`
     #' @param swagger Deprecated. See `$setDocs(docs)` or `$setApiSpec()`
-    #' @param swaggerCallback Deprecated. See `$setDocsCallback()`
+    #' @param swaggerCallback An optional single-argument function that is
+    #'   called back with the URL to an OpenAPI user interface when one becomes
+    #'   ready. If missing, defaults to `$setDocsCallback()`.
+    #' @param quiet If `TRUE`, don't print routine startup messages.
     #' @importFrom lifecycle deprecated
     run = function(
       host = '127.0.0.1',
       port = getOption('plumber.port', NULL),
       swagger = deprecated(),
       debug = deprecated(),
-      swaggerCallback = deprecated()
+      swaggerCallback,
+      quiet = FALSE
     ) {
 
       if (isTRUE(private$disable_run)) {
```

Comment:
Make sure the dots are empty

```suggestion
      ellipsis::check_dots_empty()
      
      if (isTRUE(private$disable_run)) {
```